.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -231,7 +235,7 @@
 
       #if ( $isUmbrellaCategory )
       <div class="secondary-logo">
-        <a class="branded-logo" href="${umbrellaHref}">${umbrellaBrandLabel}</a>
+        <a class="branded-logo" href="${umbrellaDirectoryPath}">${umbrellaBrandLabel}</a>
```

Comment:
This might cause WAVE problems, I will be able to check when I deploy to dev cascade